tests/transports-governance/test_governance_hierarchical_validation.py

```
@@ -0,0 +1,895 @@
+
+        # Concurrent attempts to violate hierarchy
+        def increase_team_budget():
+            return governance_client.update_budget(team_budget_id, {"max_limit": 15000})
+
+        def decrease_customer_budget():
+            return governance_client.update_budget(
+                customer_budget_id, {"max_limit": 3000}
+            )
+
+        # Both operations would violate hierarchy - only one should succeed
+        results = concurrency_helper.run_concurrent_requests(
```

Ruff (0.12.2)

Local variable `results` is assigned to but never used

Remove assignment to unused variable `results`

(F841)